server/apps/scaling-pulsar-smtp/README.md

```diff
@@ -0,0 +1,70 @@
+# Pulsar backed SMTP processing
+
+## App goals
+
+Ultimately this app will provide an easily scalable smtp processing module based on Apache Pulsar for small to large deployments. The dependency on cassandra is temporary and will be removed incrementally as suitable alternative implementations for the guice modules are integrated.
+
+Cassandra is pretty expensive to operate and only makes sense for really large scale deployments. On the other hand, there are cost-efficient SAAS deployments of pulsar. For small scale deployments destined to grow this app will instead rely on 
+- A blobstore (for mail repository) 
+- A PostgresSQL database for reference data (domain list, recipient rewrite table and users)
+
+The idea being that it will be easy enough to operate this assembly at a "reasonable" cost in the cloud until the scale is large enough to warrant switching to the fully distributed app. 
+
+## Building
+
+In order to build both the ZIP and docker packaging, run:
```

Comment:
As far as I am aware of there is no ZIP packaging in place for this

Reply:
fixedup